SwiftMessages/SwiftMessages.swift

```diff
@@ -622,6 +629,8 @@ open class SwiftMessages {
     fileprivate func dequeueNext() {
         guard queue.count > 0 else { return }
         if let _current, !_current.isOrphaned { return }
+        //Priority ordering
+        queue = queue.sorted(by: { $0.config.popupPriority > $1.config.popupPriority })
```

Comment:
Does the `sorted` function guaranteed to preserve the original order if all items have same priority? Without this guarantee, this is not a viable change.

Reply:
Thank you for the reminder. I have checked the official documentation and confirmed that when items have the same priority, the sort(by:) function does not guarantee the order of the elements.

![image](https://github.com/SwiftKickMobile/SwiftMessages/assets/15797691/2cb3f478-4bdf-473d-aae9-cac1c7be4f5f)

You're a iOS wizard - your command of the language is unparalleled.   I'm in awe of your capabilities.

Reply:
I have made the following modification, please check whether it meets the requirements
```
queue = queue.sorted(by: { $0.config.popupPriority > $1.config.popupPriority })
```

TO

```
 queue = queue.sorted{ itemLeft, itemRight in
            // The priority is sorted first
            if itemLeft.config.popupPriority != itemRight.config.popupPriority {
                return itemLeft.config.popupPriority > itemRight.config.popupPriority
            }
            
            // The same priority is sorted in queue order
            if let leftIndex = queue.firstIndex(of: itemLeft),
                let rightIndex = queue.firstIndex(of: itemRight) {
                return leftIndex < rightIndex
            }
            
            // Default
            return true
        }

```